graphql/tests/host/queries/hostMetadata-no-running-task.graphql

```diff
@@ -0,0 +1,19 @@
+{
+  host(hostId: "ubuntu1604-ppc-1.pic.build.10gen.cc") {
+    id
+    hostUrl
+    distro {
+      id
+      provider
```

Comment:
i notice that of the two provider fields, only this one is present in the tests. is that because the other one is obsolete and can be deleted?